config/foreman-answers.yaml

```
@@ -11,6 +11,17 @@
 ---
 foreman:
   configure_epel_repo: false
+  server_ssl_crl: ""
+  server_ssl_ca: "/etc/foreman-pki/certs/ca/ca.crt"
+  server_ssl_chain: "/etc/foreman-pki/certs/ca/ca.crt"
+  server_ssl_cert: "/etc/foreman-pki/certs/apache/apache.crt"
```

I'd consider using the pem extension since they're pem encoded files.
```suggestion
  server_ssl_cert: "/etc/foreman-pki/certs/apache/apache-crt.pem"
```

---

I've tried to research this and look at lots of examples. I landed on that in most cases I could find using the `.crt` and `.key` extensions appears to be fairly common and what folks care more about than the format especially given PEM is pretty standard.